### PR TITLE
Final documentation refinement

### PR DESCRIPTION
**Pull request by wojpon**
To finish ourinternship we need to provide clear and consistent documentation which covers the whole project.

Changes:

Fixed links, changed the place of some links, added sequence diagram.

### CODE REVIEW

File: docs/github-connector/configuration.md [hunk 1]
```diff
@@ -2,25 +2,32 @@
...
 
 ## Prerequisites
 
 - Kyma with the GitHub Connector [installed](/docs/github-connector/installation.md)
-- Connection to Kyma
+- Connection to Kyma Console
 
 ## Installation
 
 1. Find the newly created GitHub Connector application and [bind it to the namespace](https://kyma-project.io/docs/components/application-connector/#tutorials-bind-an-application-to-a-namespace) of your choice.
```
Comment (colunira) [inline docs/github-connector/configuration.md]: I don't think this is a part of creating a webhook.

File: docs/github-connector/README.md [hunk 2]
```diff
@@ -2,14 +2,9 @@
...
-
-- new pull request
-- new pull request review
-- new issue
+The GitHub Connector is a tool to Kyma for handling GitHub events. It registers GitHub API and events in the Application Registry. Then, it converts events incoming from connected GitHub webhooks to the format acceptable by Kyma and forwards them to the Event Bus. GitHub Connector handles all GitHub events.
 
 ## Table of Contents
 
 - [Installation](installation.md)
-- [Configuration](configuration.md)
```
Comment (colunira) [inline docs/github-connector/README.md]: Why you delete the link to configuration?

File: docs/github-connector/configuration.md [hunk 3]
```diff
@@ -2,25 +2,32 @@
...
+
+```shell
+kubectl get deployments -n {NAMESPACE} {DEPLOYMENT_NAME} -o jsonpath='{.spec.template.spec.containers[0].env[3].value}'
+```
+
+5. Set other fields as follows:
 
     - **Content type**: `application/json`
-    - **Secret**: `my-secret-key`
+    - **Secret**: previously obtained secret (nie wiem co z tym zrobic)
```
Comment (colunira) [inline docs/github-connector/configuration.md]: As you explained above how to get secret you could delete this line.

File: docs/github-connector/configuration.md [hunk 4]
```diff
@@ -2,25 +2,32 @@
...
     - **SSL verification**: `Disabled`
 
-    >**NOTE:** Secret is defined statically in code and SSL verification is disabled.
+    >**NOTE:** If your Kyma cluster is not SSL secured, SSL verification will not be working.
```
Comment (colunira) [inline docs/github-connector/configuration.md]: You say that SSL verification has to be disabled, then say it won't work if your cluster is not SSL secured. It's misleading. Specify if user can or cannot turn on the verification

File: docs/github-connector/installation.md [hunk 5]
```diff
@@ -83,4 +83,4 @@ To correctly remove all GitHub Connector resources, you must delete them in orde
 
     >**NOTE:** To define the Namespace in which to install chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To crate a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
 
-3. For further steps, see [configuration page](/docs/github-connector/configuration.md).
+3. For further steps, see [manulal configuration page](/docs/github-connector/configuration.md).
```
Comment (colunira) [inline docs/github-connector/installation.md]: Manulal? ;) 
Also, I wouldn't say that adding a webhook manually is connectors configuration.

File: docs/github-connector/installation.md [hunk 6]
```diff
@@ -83,4 +83,4 @@ To correctly remove all GitHub Connector resources, you must delete them in orde
 
     >**NOTE:** To define the Namespace in which to install chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To crate a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
    >**NOTE:** To define the Namespace in which to install the chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To create a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
```

File: docs/slack-connector/README.md [hunk 7]
```diff
@@ -7,4 +7,4 @@ The Slack Connector is an additional tool to Kyma for handling communication wit
 ## Table of Contents
```
Comment (majakurcius) [inline docs/slack-connector/README.md]: **Line 5:**
  - I'd say `an additional tool for Kyma` or `an addon to Kyma`
  - `Slack API` -> `the Slack API`

File: addons/githubconnector-1.0.0/docs/overview.md [hunk 8]
```diff
@@ -1,21 +1,19 @@
...
+This addon allows you to communicate with GitHub. You can handle events incoming from GitHub repositories or manage repositories through the GitHub API. You must provision an instance for every repository you want to communicate with.
 
 ## Installation
 
-1. Provision this addon. Plans' and fields' meaning is explained below.
+1. Provision the GitHub Connector addon. Plans' and fields' meaning is explained below.
 2. Go to `Service Management > Catalog > Services`. Find a service named `github-{REPOSITORY-NAME}` and add it.
 
-Now you can start using the GitHub Connector. Add new event trigger to react to chosen GitHub notifications or bind this service in lambda to send authorized request to GitHub API.
+Now you can start using the GitHub Connector. Add new event trigger to react to chosen GitHub notifications or bind this service in lambda to send authorized request to the GitHub API.
```
Comment (majakurcius) [inline addons/githubconnector-1.0.0/docs/overview.md]: `new event trigger` -> `a new Event trigger` 
`in lambda` -> `in a lambda`

File: addons/slackconnector-1.0.0/docs/overview.md [hunk 9]
```diff
@@ -1,29 +1,25 @@
...
+With the use of the token provided during the provisioning of the addon, the Slack Connector allows for sending requests to the Slack Web API. The requests are specified during the installation of the Slack Application to the workspace. The example requests are: posting a message to a specified channel, getting the list of the current users, etc.
 
 ## Installation
 
 1. [Provision](#provisioning) this addon.
-2. Go to `Service Management > Catalog > Services`. Find a service named `slack-connector-{WORKSPACE-NAME}` and add it.
+2. In **Service Management**, go to **Catalog** and choose **Services**. Find the service named `slack-connector-{DESIRED_WORKSPACE_NAME}` and add it.
 
-Now you can start using the Slack Connector. Add channel ID to lambda environmental variables to send authorized request to Slack Web API.
+Now you can start using the Slack Connector. Add channel name to lambda environmental variables to send authorized requests to the Slack Web API.
```
Comment (majakurcius) [inline addons/slackconnector-1.0.0/docs/overview.md]:
```suggestion
Now you can start using the Slack Connector. Add the channel name to the lambda environmental variables to send authorized requests to the Slack Web API.
```

File: docs/github-connector/installation.md [hunk 10]
```diff
@@ -13,43 +13,37 @@
...
-The GitHub Connector is a component which allows interaction with the GitHub API from inside of Kyma environment. The simplest way to install the GitHub Connector in Kyma is to install it as an Add-On.
+The GitHub Connector is a component which allows interaction with the GitHub API from inside of Kyma environment. The simplest way to install the GitHub Connector in Kyma is to install it as an addon.
 
-## Installation in Kyma as an Add-On
+## Installation in Kyma as an Addon
 
 ### Prerequisites
 
-- GitHub App with desired privileges installed to the destination repository or organization. To create a new application, go [here](https://github.com/settings/apps) or access **Github Apps** in the account through **Settings** in **Developer settings**.
+- Personal Acces Token with desired privileges. To generate a new token, go [here](https://github.com/settings/tokens) or access **Personal access tokens** in the account through **Developer settings** in **Settings**.
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
- Personal Access Token with desired privileges. To generate a new token, go [here](https://github.com/settings/tokens) or access **Personal access tokens** in the account through **Developer settings** in **Settings**.
```

File: docs/github-connector/installation.md [hunk 11]
```diff
@@ -81,6 +76,6 @@ To correctly remove all GitHub Connector resources, you must delete them in orde
 
     >**CAUTION:** Make sure the Kyma address is in the correct format. It consists of the domain name and cannot begin with the dot. For example, `35.187.32.214.xip.io`.
 
-    >**NOTE:** To define the Namespace in which to install chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To crate a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
+    >**NOTE:** To define the Namespace in which to install the chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To create a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
 
-3. For further steps, see [configuration page](/docs/github-connector/configuration.md).
+If you want to add new repository after the instalation, see [manual configuration page](/docs/github-connector/manual_connection.md).
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
If you want to add new repository after the installation, see the [manual configuration page](/docs/github-connector/manual_connection.md).
```

File: docs/github-connector/manual_connection.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+# Connecting a new GitHub repository manually
+
+## Overview
+
+This document describes how to manually connect a GitHub repository to the existing GitHub Connector installed in Kyma the enviroment.
```
Comment (majakurcius) [inline docs/github-connector/manual_connection.md]:
```suggestion
This document describes how to manually connect a GitHub repository to the existing GitHub Connector installed in the Kym environment.
```

File: docs/github-connector/manual_connection.md [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+   
+  > **NOTE:**  You can find the exposed service URL in the Kyma Console in your Namespace's APIs.
+
+3. To get the Secret, which is required during the webhook setup, use this command:
+
+```shell
+kubectl get deployments -n {NAMESPACE} {DEPLOYMENT_NAME} -o jsonpath='{.spec.template.spec.containers[0].env[3].value}'
+```
+
+4. Set other fields as follows:
```
Comment (majakurcius) [inline docs/github-connector/manual_connection.md]:
```suggestion
4. Set the other fields as follows:
```

**Comment by majakurcius** [comment]
Please adjust capitalization for Application(s)/application(s), Namespace(s), and Event(s).

File: docs/azure-comments-analytics/README.md [hunk 14]
```diff
@@ -2,15 +2,13 @@
...
-
-This add-on allows you to install the scenario provided by Team Flying Seals. Azure Comments Analytic receives information about Github's Issue from the Github Connector. Next, the lambda function analyses it using the Azure Broker and then, if the Issue's sentiment is negative, the lambda sends it to Slack and labels the Issue on Github.
+This addon allows you to install an example scenario. Azure Comments Analytic receives information about Github's Issue from the Github Connector. Next, the lambda function analyses it using the Azure Broker and then, if the Issue's sentiment is negative, the lambda sends it to Slack and labels the Issue on Github.
 
 ## Installation
 
-1. Provision [Github Connector](https://github.com/kyma-incubator/hack-showcase/blob/master/docs/github-connector/README.md).
-2. Provision [Slack Connector](https://github.com/kyma-incubator/hack-showcase/blob/master/docs/slack-connector/README.md).
-3. Provision [Azure Broker](https://github.com/kyma-project/addons/tree/master/addons/azure-service-broker-0.0.1).
+1. Provision [The Github Connector](https://github.com/kyma-incubator/hack-showcase/blob/master/docs/github-connector/README.md).
```
Comment (majakurcius) [inline docs/azure-comments-analytics/README.md]:
```suggestion
1. Provision [the Github Connector](https://github.com/kyma-incubator/hack-showcase/blob/master/docs/github-connector/README.md).
```

File: docs/azure-comments-analytics/README.md [hunk 15]
```diff
@@ -2,15 +2,13 @@
...
+2. Provision [The Slack Connector](https://github.com/kyma-incubator/hack-showcase/blob/master/docs/slack-connector/README.md).
```
Comment (majakurcius) [inline docs/azure-comments-analytics/README.md]:
```suggestion
2. Provision [the Slack Connector](https://github.com/kyma-incubator/hack-showcase/blob/master/docs/slack-connector/README.md).
```

File: docs/azure-comments-analytics/README.md [hunk 16]
```diff
@@ -2,15 +2,13 @@
...
+3. Provision [The Azure Broker](https://github.com/kyma-project/addons/tree/master/addons/azure-service-broker-0.0.1).
```
Comment (majakurcius) [inline docs/azure-comments-analytics/README.md]:
```suggestion
3. Provision [the Azure Broker](https://github.com/kyma-project/addons/tree/master/addons/azure-service-broker-0.0.1).
```

File: docs/github-connector/installation.md [hunk 17]
```diff
@@ -81,6 +76,6 @@ To correctly remove all GitHub Connector resources, you must delete them in orde
 
     >**CAUTION:** Make sure the Kyma address is in the correct format. It consists of the domain name and cannot begin with the dot. For example, `35.187.32.214.xip.io`.
 
-    >**NOTE:** To define the Namespace in which to install chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To crate a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
+    >**NOTE:** To define the Namespace in which to install the chart, add the `--namespace` flag to the command. To define the GitHub URL, add the `--set githubURL` flag. To create a webhook on one repository, use the construction `repos/:owner/:repo`. To create a webhook on the whole organization, use `orgs/:org`. To provide the security token, use the `--set githubToken` flag.
 
-3. For further steps, see [configuration page](/docs/github-connector/configuration.md).
+If you want to add new repository after the installation, see the [manual configuration page](/docs/github-connector/manual_connection.md).
```
Comment (majakurcius) [inline docs/github-connector/installation.md]:
```suggestion
If you want to add a new repository after the installation, see the [manual configuration page](/docs/github-connector/manual_connection.md).
```

**Pull request merged**